### DISCUSSION TITLE
Is it possible to make Ruff ignore specific lines globally by pattern matching or something?

### DISCUSSION THREAD

**Discussion by Lassii-** [astral-sh/ruff#6088] [Q&A]
Hello,

I'm working on a project where we have certain types of lines we'd like to go past 120 characters to improve readability but otherwise enforce the 120 character limit with Ruff. Is there some way to make Ruff ignore lines starting with a specific pattern when linting? I know you can add a comment to disable specific lines but there's lots of these that would need to be ignored and having to manually add the comment to each new line of this type would be an additional developer burden and thing to remember.

In short I'd like to set a single parameter in Ruff configuration in style of "for lines starting with this pattern ignore the line length requirement".

Thank you for your time.

**Comment by e10v** [comment]
Agree, this feature would be very useful. My use case is empty lines with one space after the prompt: ">>> ". This triggers the rule [W291](https://docs.astral.sh/ruff/rules/trailing-whitespace/#trailing-whitespace-w291). And I don't want to disable this rule for other lines in the docstring.

UPD. It could be a config similar to [tool.coverage.report.exclude_lines](https://coverage.readthedocs.io/en/7.6.10/config.html#report-exclude-lines).

**Comment by dhruvmanila** [comment]
Related: https://github.com/astral-sh/ruff/issues/8687
